Linker error in debug build on Windows

I tried building a debug version of typed-ast on Windows (Python 3.7.5, Visual Studio 2015):
```cmd
> cd D:\Repos\github.com\python\typed_ast
> c:\Python37\python_d.exe setup.py build --debug
```

I first get a warning about inconsistent DLL linkage
```cmd
D:\Repos\github.com\python\typed_ast\ast27\Parser\parser.c(18): warning C4273: 'Py_DebugFlag': inconsistent dll linkage
c:\python37\include\pydebug.h(10): note: see previous definition of 'Py_DebugFlag'
```

and then a linker error:
```cmd
Creating library build\temp.win-amd64-3.7-pydebug\Debug\ast27/Parser\_ast27_d.cp37-win_amd64.lib and object build\temp.win-amd64-3.7-pydebug\Debug\ast27/Parser\_ast27_d.cp37-win_amd64.exp
parser.obj : error LNK2001: unresolved external symbol Py_DebugFlag
build\lib.win-amd64-3.7-pydebug\typed_ast\_ast27_d.cp37-win_amd64.pyd : fatal error LNK1120: 1 unresolved externals
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\amd64\\link.exe' failed with exit status 1120
```

(this also happens with Visual Studio 2019)

I was able to fix it by changing the line
```python
extern int Py_DebugFlag;
```

to
```python
PyAPI_DATA(int) Py_DebugFlag;
```

in `ast27/Parser/parser.c` and `ast3/Parser/parser.c`. However I am not sure if that is the correct approach as I have not a lot of knowledge about the CPython internals.

--- a/ast27/Parser/parser.c
+++ b/ast27/Parser/parser.c
@@ -15,7 +15,7 @@
 
 
 #ifdef Py_DEBUG
-extern int Py_DebugFlag;
+PyAPI_DATA(int) Py_DebugFlag;
 #define D(x) if (!Py_DebugFlag); else x
 #else
 #define D(x)

--- a/ast3/Parser/parser.c
+++ b/ast3/Parser/parser.c
@@ -15,7 +15,7 @@
 
 
 #ifdef Py_DEBUG
-extern int Py_DebugFlag;
+PyAPI_DATA(int) Py_DebugFlag;
 #define D(x) if (!Py_DebugFlag); else x
 #else
 #define D(x)

